core/storage/leveldb/leveldb.cpp

```diff
@@ -69,6 +69,15 @@ namespace kagome::storage {
     return error_as_result<void>(status, logger_);
   }
 
+  outcome::result<void> LevelDB::put(const Buffer &key, Buffer &&value) {
```

Comment:
if make_slice() produces new object, I don't see how && helps. const reference is enough.

Reply:
Yes, `make_slice` accepts its argument by a constref, so an rvalue is not needed here

Reply:
Interface requires.